cmd/opentelemetry-collector/app/exporter/elasticsearch/factory.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package elasticsearch
+
+import (
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configerror"
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+	"github.com/open-telemetry/opentelemetry-collector/exporter"
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/es"
+)
+
+const (
+	// TypeStr defines type of the Elasticsearch exporter.
+	TypeStr = "jaeger_elasticsearch"
+)
+
+// OptionsFactory returns initialized es.OptionsFactory structure.
+type OptionsFactory func() *es.Options
+
+// DefaultOptions creates Elasticsearch options supported by this exporter.
+func DefaultOptions() *es.Options {
+	return es.NewOptions("es")
```

Comment:
It has to be `es` because we want to be backward compatible with the configuration with jaeger collector. This name is used in flag names e.g. `--es.server-urls=`